### PR TITLE
Add dappctr config option to write logs to file.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-402.

### CODE REVIEW

File: dappctrl-test.config.json [hunk 1]
```diff
@@ -148,7 +148,8 @@
     },
 
     "Log": {
-        "Level": "info"
+        "Level": "info",
+        "Logpath": ""
```
Comment (ababo) [inline dappctrl-test.config.json]: "LogPath"

File: util/log.go [hunk 2]
```diff
@@ -57,17 +64,55 @@ func parseLogLevel(lvl string) int {
...
+func createLogFile(path string) (*os.File, error) {
+	info, err := os.Stat(path)
+	if err != nil {
+		return nil, err
+	}
+	if !info.IsDir() {
+		return nil, err
+	}
+
+	fileName := gofmt.Sprintf("dappctrl-%d.log", time.Now().Unix())
```
Comment (ababo) [inline util/log.go]: I believe it's better to make more explicit timestamps as part of the names, e.g. `dappctrl-2018-06-21.log`. Much easier to use.

File: util/log.go [hunk 3]
```diff
@@ -57,17 +64,55 @@ func parseLogLevel(lvl string) int {
...
 	}
 
-	return &Logger{
-		logger: log.New(os.Stderr, "", log.LstdFlags),
-		level:  lvl,
-	}, nil
+	logger := &Logger{level: lvl}
+
+	if conf.Logpath != "" {
+		file, err := createLogFile(conf.Logpath)
```
Comment (ababo) [inline util/log.go]: Why do we need to save file instead or redirecting STDERR? You don't implement rotation, so this is almost equivalent.

Comment (dzeckelev) [reply util/log.go]: This will be solved in another task, when a log rotation will be implemented.

**Comment by ababo** [comment]
If we're going to make a full-fledged logger with file rotation I suggest to choose an existing logger to work under the `util.Logger` hood. Indeed it's not so trivial to implement such rotation properly. Otherwise I don't see any sense to support explicit file output while having output to STDERR which can be easily redirected.

**Pull request merged**